echopype/convert/add_ancillary.py

```diff
@@ -99,11 +99,11 @@ def mapping_get_multiple(mapping, keys, default=None):
                             default=np.full(num_obs, np.nan),
                         ),
                     ),
-                    "water_level": (
+                    "range_offset": (
```

Comment:
FYI I've removed `add_ancillary.py` in PR #574, now merged. It turns out it was an obsolete remnant.